app/views/internal/configs/show.html.erb

```diff
@@ -88,14 +88,19 @@
                                 placeholder: "email@staff.com" %>
               <div class="alert alert-info">Email of the staff account</div>
             </div>
-
             <div class="form-group">
-              <%= f.label :social_networks_handle %>
-              <%= f.text_field :social_networks_handle,
-                               class: "form-control",
-                               value: SiteConfig.social_networks_handle,
-                               placeholder: "thepracticaldev" %>
-              <div class="alert alert-info">Social networks handle (eg. Twitter, GitHub)</div>
+              <%= f.fields_for :social_media_handles do |ff| %>
```

Comment:
I would suggest a more clear name instead of `ff` but couldn't invent one for now :D